src/Microsoft.DotNet.Build.Tasks.Packaging/src.Desktop/project.json

```diff
@@ -4,7 +4,7 @@
     "Newtonsoft.Json": "7.0.1",
```

Comment:
Does updating to the latest Platforms package in this project fix the error you are seeing?